libs/@local/hash-validation/src/lib.rs

```diff
@@ -334,11 +337,48 @@ mod tests {
     ) -> Result<PropertyWithMetadataObject, Report<[TraversalError]>> {
         install_error_stack_hooks();
 
+        let mut ontology_type_resolver = OntologyTypeResolver::default();
+        let entity_types = entity_types
+            .into_iter()
+            .map(|entity_type| {
+                let entity_type = serde_json::from_str::<EntityType>(entity_type)
+                    .expect("failed to parse entity type");
+                let entity_type_id = EntityTypeUuid::from_url(&entity_type.id);
+                ontology_type_resolver
+                    .add_unresolved_entity_type(entity_type_id, Arc::new(entity_type.clone()));
+                (entity_type_id, entity_type)
+            })
+            .collect::<Vec<_>>();
+
+        let entity_type =
+            serde_json::from_str::<EntityType>(entity_type).expect("failed to parse entity type");
+        let entity_type_uuid = EntityTypeUuid::from_url(&entity_type.id);
+        ontology_type_resolver
+            .add_unresolved_entity_type(entity_type_uuid, Arc::new(entity_type.clone()));
+
+        let resolved_data = ontology_type_resolver
+            .resolve_entity_type_metadata(entity_type_uuid)
+            .expect("entity type not resolved");
+        let closed_entity_type = ClosedEntityType::from_resolve_data(entity_type, &resolved_data)
+            .expect("Could not close church");
```

Comment:
Is this supposed to say 'church'?

Reply:
That's a copy-paste error (we had a `church` and a `building` entity type elsewhere).
```suggestion
            .expect("Could not close entity type");
```